dataland-frontend/tests/e2e/utils/GeneralApiUtils.ts

```diff
@@ -170,5 +170,6 @@ export function submitFilledInEuTaxonomyForm(
   cy.wait(`@${postRequestAlias}`, { timeout: Cypress.env("long_timeout_in_ms") as number }).then((interception) => {
     expect(interception.response?.statusCode).to.eq(200);
   });
+  cy.wait(4000);
```

Comment:
Can we remove this?
We generally want to avoid hardcoded waits, unless there is absolutetly no other way to make a test stable.

Do you think it is possible to remove it? Have you experienced problem if the `cy.wait` statement is not there?

Reply:
I'll remove it. I added it because locally it was sometimes failing on the next line `cy.contains("td", "EU Taxonomy");` .. but locally only, I think.

Reply:
👍